test/format/wts.c

```diff
@@ -365,14 +365,30 @@ wts_init(void)
 	 * Configure the maximum key/value sizes, but leave it as the default
 	 * if we come up with something crazy.
 	 */
+	/* XXX disable overflow records.  Test-only. */
+#define	__XXX_NO_OVERFLOW 0
```

Comment:
@keithbostic you'll want to change this to 1 to repro the overflow issues.  This will all disappear before this branch is ready for final review.